docker-compose.yml

```diff
@@ -82,7 +82,7 @@ services:
   dataland-edc:
     restart: always
     profiles: ["production"]
-    image: ghcr.io/d-fine/datalandedc/edcdummyserver:latest
+    image: ghcr.io/d-fine/datalandedc/edcdummyserver:RELEASE-0.0.4
```

Comment:
Muss hier nicht das Image vom "echten" EDC verwendet werden?
Oder wollen wir in der production den dummy benutzen?